src/sbvr-api/permissions.ts

```diff
@@ -1273,6 +1273,102 @@ export const getUserPermissions = async (
 	}
 };
 
+export const getRolePermissions = async (
+	roleName: string,
+	userId: number,
+	tx?: Tx,
+): Promise<string[]> => {
+	try {
+		if (typeof userId === 'string') {
+			userId = parseInt(userId, 10);
+		}
+		if (!Number.isFinite(userId)) {
+			throw new Error(`User ID has to be numeric, got: ${typeof userId}`);
+		}
+		return await $getRolePermissions(roleName, userId, tx);
+	} catch (err: any) {
```

Comment:
```suggestion
	} catch (err: unknown) {
```
Technically I think its more type safe to use unknown instead of any in this case.